templates/scenarios/js/m365-tab/teamsapp.local.yml.tpl

```diff
@@ -5,15 +5,15 @@ version: 1.0.0
 provision:
   - uses: aadApp/create # Creates a new AAD app to authenticate users if AAD_APP_CLIENT_ID environment variable is empty
```

Comment:
should we use aadApp/createIfNotExists?

Reply:
after discussed with Tim/John, will keep using `aadApp/create`